test/defaultResolver.js

```diff
@@ -3,10 +3,14 @@ var r = require("./../resolver.js")
 
 describe('resolver.defaultResolver', function() {
     it('nothing', function() {
-        assert.strictEqual(
-            r.defaultResolver('', '', {headers:{}}),
-            null
-        );
+        return r.defaultResolver('', '', {headers:{}})
```

Comment:
Unfortunately this is the only test we have for the default resolver which means we only ever test the first lines.